packages/notification/src/components/GenericNotification.tsx

```diff
@@ -30,6 +30,7 @@ type Props = {
   noBottomMargin?: boolean
   forceMultiline?: boolean
   headingProps?: HeadingProps
+  isSubtle?: boolean
```

Comment:
Since we want it confined, I'd actually propose we add `classNameOverride` here and add the class to the InlineNotification instead.

If we later find that we want it for the other notifications, we will move it then.